packages/fast-components-react-base/src/card/card.schema.json

```diff
@@ -3,7 +3,7 @@
     "title": "Card",
     "description": "A card component's schema definition.",
     "type": "object",
-    "id": "card",
```

Comment:
@chrisdholt @janechu For components that do not have MSFT versions (simply wrapped in the JSSManager), we re-use the "base" schema - is that going to be an issue?

Reply:
Ah - if we are trying to reference both a styled and unstyled instance within a tooling site, I think this would be an issue. This would likely pull in an unstyled base instance.

@janechu @nicholasrice what are your thoughts on managing duplicate schemas?

Reply:
I think we should copy schemas over and provide duplicates with the precise ids for the package and we should run our examples through tv4 to validate our example data against the schemas, in case it ever goes out of date. Which incidentally, we should be doing regardless but this would help validate our APIs.